src/sentry/tasks/relay.py

```diff
@@ -3,6 +3,7 @@
 import sentry_sdk
 from django.conf import settings
 
+import sentry.options
```

Comment:
```suggestion
from sentry import options
```

Reply:
i like it this way, and there is no consistency in the codebase anyway...

Reply:
apparently there is consistency, just not from the one place i copied this from.  fixed them both